src/app/index.route.coffee

```diff
@@ -134,6 +134,7 @@
         label: 'mnoe_admin_panel.dashboard.sub_tenants.title'
       resolve:
         skip: (MnoeCurrentUser) -> MnoeCurrentUser.skipIfNotAdminRole(['admin'])
+        skipCondition: (RoutingHelper, SUB_TENANT_CONFIG) -> RoutingHelper.skipUnlessCondition(SUB_TENANT_CONFIG && SUB_TENANT_CONFIG.enabled)
```

Comment:
Wrap the constant in the adminConfig service. This help with providing default value.
`adminConfig.isSubTenantEnabled()`